pkg/file/seekjoiner/joiner_test.go

```diff
@@ -114,3 +166,62 @@ func TestJoinerWithReference(t *testing.T) {
 		t.Fatalf("expected resultbuffer %v, got %v", resultBuffer, firstChunk.Data()[:len(resultBuffer)])
 	}
 }
+
+func TestEncryptDecrypt(t *testing.T) {
+	var tests = []struct {
+		chunkLength int
+	}{
+		{10},
+		{100},
+		{1000},
+		{4095},
+		{4096},
+		{4097},
+		{1000000},
+	}
+
+	for _, tt := range tests {
+		t.Run(fmt.Sprintf("Encrypt %d bytes", tt.chunkLength), func(t *testing.T) {
+			store := mock.NewStorer()
+			joinner := seekjoiner.NewSimpleJoiner(store)
+
+			g := mockbytes.New(0, mockbytes.MockTypeStandard).WithModulus(255)
+			testData, err := g.SequentialBytes(tt.chunkLength)
+			if err != nil {
+				t.Fatal(err)
+			}
+			ctx := context.Background()
+			pipe := pipeline.NewPipelineBuilder(ctx, store, storage.ModePutUpload, true)
```

Comment:
this true means you encrypt the data right? maybe worth a comment

Reply:
which comment? the test name describes that it tests for encryption, and the bool parameter in NewPipelineBuilder is called `encrypt` do we need more explicit comments here?